mlir/lib/Dialect/Vector/Transforms/VectorTransforms.cpp

```diff
@@ -1103,6 +1103,127 @@ class ExtractOpFromElementwise final
   }
 };
 
+/// Pattern to rewrite vector.extract(vector.load) -> vector/memref.load.
+/// ```
+///  vector.load %arg0[%arg1] : memref<?xf32>, vector<4xf32>
+///  vector.extract %0[1] : f32 from vector<4xf32>
+/// ```
+/// Gets converted to:
```

Comment:
Here and in other places, to improve the flow:
```suggestion
/// Pattern to rewrite vector.extract(vector.load) -> vector/memref.load.
///
/// Example:
/// ```
///  vector.load %arg0[%arg1] : memref<?xf32>, vector<4xf32>
///  vector.extract %0[1] : f32 from vector<4xf32>
/// ```
/// gets converted to:
```